src/mcts/search.cc

```diff
@@ -293,10 +292,22 @@ std::vector<std::string> Search::GetVerboseStats(Node* node) const {
         << edge.GetMove(is_black_to_move).as_string();
     
     float Q = edge.GetQ(fpu, draw_score, logit_q);
-    float M_effect = do_moves_left_adjustment 
-        ? (std::clamp(m_slope * edge.GetM(0.0f), -m_cap, m_cap) *
-            std::copysign(1.0f, -Q) * (a + b * std::abs(Q) + c * Q * Q)) 
-        : 0.0f;
+    float ML_utility = 0.0f;
```

Comment:
Maybe factor out this to a separate function, as it's quite a heavy piece of code that occurs twice?

Reply:
> When the feature is completely off (MLH is ignored), is there any nps drop?
Not at all, since when `do_moves_left_adjustment == false` this code changes nothing.

Reply:
I am not sure how to do it cleanly due to the slight difference between two. Any suggestion?

Reply:
This part is removed after #1268, so it does not matter now.